cmd/manager/main.go

```diff
@@ -109,3 +123,20 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func setLoggerForProduction() {
+	sink := zapcore.AddSync(os.Stderr)
+	var opts []zap.Option
+	encCfg := zap.NewProductionEncoderConfig()
+	enc := zapcore.NewJSONEncoder(encCfg)
+	lvl := zap.NewAtomicLevelAt(zap.WarnLevel)
```

Comment:
This will essentially disable logging (only errors will be logged). Do we want to go that far?

We may prefer to log at the info level so long as it's not so verbose that it kills performance. If it is, we should also consider dropping problematic log lines.

Getting rid of info logs may make production issues hard to debug at the teething stage of the process.

As a bonus, switching to Info, means we can just use `ZapLogger(false)` and drop these lines.

Reply:
I see now that this is a user request. I would suggest adding a --log-level flag that accepts either an integer or the string value of the minimum log level to report.